tests/jeos/prepare_firstboot.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Enable jeos-firstboot as required by openQA testsuite
+# Maintainer: Guillaume GARDET <guillaume@opensuse.org>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    # Login with default credentials (root:linux)
+    assert_screen('linux-login', 300);
+    type_string("root\n");
```

Comment:
would you mind to change the code to look somehow like that?
-    type_string("root\n");
-    type_string("linux\n");
+    type_string("root\n", wait_still_screen => 5);
+    type_string("linux\n", wait_still_screen => 5);

I'm using this code in aarch64 emulation mode on x86 and it fails because it it types 'root\n' and then immediately 'linux\n' but password prompt is not ready yet and it recieves only 'inux\n' part of the string. Well it is just 10s in total so it doesn't hurt.

I guess also that on real Rpi it would be bit slower then on our powerful aarch64 KVM host.

(reference http://open.qa/api/testapi/#_type_string)

Reply:
Done.